mysql_pubkey.asc

```diff
@@ -0,0 +1,49 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
Was this file included intentionally?

Reply:
Yes, as it must be imported via apt-key to pull from the mysql apt repo. The key could be pulled from the MIT key server every time instead as described in their [docs](https://dev.mysql.com/doc/mysql-apt-repo-quick-guide/en/#repo-qg-apt-repo-manual-setup), however I've committed it because pgp.mit.edu was unavailable at some point yesterday while I was testing this (and very slow to pull from when it was available).

We can get rid of this in the future likely by switching off debian entirely to something like ubuntu that provides the mysql libmysqlclient on their apt servers.